### PR TITLE
Feat/data generation script 2.0

### PR DESCRIPTION
**Pull request by tiffaphm**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-connect.db()
```
Comment (ryanakiyama) [inline database/index.js]: File is empty, generally shouldn't merge empty files

Cut another branch for database/index.js then make another pull request when completed

Comment (tiffaphm) [reply database/index.js]: Since this file is already committed, will just leave it here. Noted for next time.
